src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -224,10 +228,58 @@ public async Task<List<SubmittedApplicationAnswer>> ExtractAnswersForApplication
                     }
                 }
             }
+            //Extract QuestionId Y0-110 answer even if it is inactive, there is an issue setting answer active to false while adding orgnisation type answer to the tabular list as the last entry
+            var hasPartnershipAnswers = answers.Any(answer => answer.QuestionId == QuestionIdPartnership);
+            if(!hasPartnershipAnswers)
+            {
+                await ExtractPartnershipAnwers(applicationId, answers);
```

Comment:
We only want to do this in certain case, where is that case defined?

Reply:
This is the condition to check if PartnershipAnswers are already extracted or not

Reply:
Ravi's response below clarifies there is no need for extra checks. 
> extra check is not required, when partnership>>add people then go back and changed to sole trade route, though that answers are not cleared at partnership question, but GetAnswerByTag returns empty, so we are not adding not relevant information.